regions/shapes/tests/test_rectangle.py

```diff
@@ -1,26 +1,52 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
+from numpy.testing import assert_allclose
+from astropy.tests.helper import pytest
 import astropy.units as u
 from astropy.coordinates import SkyCoord
 from ...core import PixCoord
 from ..rectangle import RectanglePixelRegion, RectangleSkyRegion
-from .utils import ASTROPY_LT_13
+from .utils import ASTROPY_LT_13, HAS_MATPLOTLIB
 
 
-def test_ellipse_pixel():
-    center = PixCoord(3, 4)
-    reg = RectanglePixelRegion(center, 3, 4, 5 * u.deg)
+class TestRectanglePixelRegion:
+    def setup(self):
+        center = PixCoord(3, 4)
+        self.reg = RectanglePixelRegion(
+            center=center,
+            width=4,
+            height=3,
+            angle=5 * u.deg,
+        )
 
-    assert str(reg) == 'RectanglePixelRegion\ncenter: PixCoord(x=3, y=4)\nheight: 3\nwidth: 4\nangle: 5.0 deg'
+    def test_str(self):
+        expected = 'RectanglePixelRegion\ncenter: PixCoord(x=3, y=4)\nwidth: 4\nheight: 3\nangle: 5.0 deg'
+        assert str(self.reg) == expected
 
+    @pytest.mark.skipif('not HAS_MATPLOTLIB')
+    def test_as_patch(self):
+        patch = self.reg.as_patch()
+        assert_allclose(patch.xy, (3, 4))
+        assert_allclose(patch.get_width(), 4)
+        assert_allclose(patch.get_height(), 3)
+        assert_allclose(patch._angle, 5)
```

Comment:
@cdeil I looked in the `matplotlib` source code and there's no public attribute or method (that's odd), only the `_angle` attribute.